pipeline/variables/prod.yml

```
@@ -0,0 +1,27 @@
+# Production environment variables
+variables:
+  # Environment specific
+  environmentName: 'production'
+  kubernetesNamespace: 'blog-app'
+  helmValuesFile: 'values-prod.yaml'
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)